src/runtime.ll

```diff
@@ -57,3 +58,57 @@ post:
     ret void
 }
 
+
+define %List* @dmir.intrinsic.get_list(%DMValue %list_id) alwaysinline {
+entry:
+    %id = extractvalue %DMValue %list_id, 1
+    %glob_list = load %List**, %List*** @dmir.runtime.GLOB_LIST_ARRAY, align 4
+    %array_element = getelementptr %List*, %List** %glob_list, i32 %id
+    %ret = load %List*, %List** %array_element, align 4
+
+    ret %List* %ret
+}
+
+define %DMValue @dmir.runtime.list_indexed_get(%DMValue %list_id, i32 %index) alwaysinline {
```

Comment:
Please, do not name defined inline functions `dmir.runtime.*`, use `dmir.intrinsic.*` instead